src/uct/cuda/cuda_ipc/cuda_ipc_iface.c

```diff
@@ -598,34 +650,26 @@ static UCS_CLASS_INIT_FUNC(uct_cuda_ipc_iface_t, uct_md_h md, uct_worker_h worke
         return UCS_ERR_IO_ERROR;
     }
 
-    self->streams_initialized = 0;
-    self->cuda_context        = 0;
-    ucs_queue_head_init(&self->outstanding_d2d_event_q);
-
+    kh_init_inplace(cuda_ipc_queue_desc, &self->queue_desc_map);
+    ucs_queue_head_init(&self->active_queue);
+    self->cuda_context = 0;
```

Comment:
```suggestion
    self->cuda_context = NULL;
```
Please replace by `NULL` for consistency, since we compare the field with `NULL`.

Reply:
ok